src/components/common/dataset-label.tsx

```diff
@@ -31,15 +32,12 @@ const DatasetName = styled.div.attrs({
 `;
 
 interface DatasetLabelType {
-  dataset: {
-    color?: string;
-    label?: string;
-  };
+  dataset: KeplerTable;
 }
 
 const DatasetLabel = ({dataset}: DatasetLabelType) => (
   <CenterFlexbox>
-    <DatasetSquare className="dataset-clolor" color={dataset.color} />
+    <DatasetSquare className="dataset-clolor" backgroundColor={dataset.color} />
```

Comment:
Is the rename for this prop intentional? I see the implementation was also updated, but is this desired?

Reply:
Looks like param named "color" is used by library and has different type, so there was a name conflict. And I decided that renaming is the easiest way to avoid the conflict there